pdns/ednsoptions.hh

```diff
@@ -32,14 +32,21 @@ struct EDNSOptionCode
 /* extract a specific EDNS0 option from a pointer on the beginning rdLen of the OPT RR */
 int getEDNSOption(char* optRR, size_t len, uint16_t wantedOption, char ** optionValue, size_t * optionValueSize);
 
-struct EDNSOptionView
+struct EDNSOptionViewValue
 {
   const char* content{nullptr};
   uint16_t size{0};
 };
 
+struct EDNSOptionView
```

Comment:
Did you keep a struct instead of a type alias for style/readability ?

Reply:
Mostly because otherwise it was mapped as a Lua table by LuaWrapper and I couldn't get my bindings to work.